doc/dev-guide/topolvm-csi.md

```diff
@@ -0,0 +1,35 @@
+# Topolvm CSI
+
+- LVM Operator deploys the Topolvm CSI which provides dynamic provisioning of
+  local storage.
+- Please refer to topolvm [docs][topolvm-docs] for more details on topolvm
+
+## CSI Driver
+
+- *csiDriver* reconcile unit deploys Topolvm CSIDriver resource
+- The LVM operator does not expose support for ephemeral volumes.
```

Comment:
We can remove this sentence as the latest topolvm has removed ephemeral volumes support.